### ISSUE TITLE
Add function to extract training + validation set after initial_validation_split()

### ISSUE THREAD

**Issue by paulcbauer** [tidymodels/rsample#467]
Hi, I am just coming back from teaching a tidymodels workshop (tidymodels is fun to teach!). I wonder whether it might make sense to add a function, e.g., `training_validation()` that can be used to extract the full training set at the end. That is assuming one wants to build the final model on the latter. Right now, one would do this manually I think, e.g., below:

```
data_train <- training(data_split)
data_validation <- validation(data_split)
data_test <- testing(data_split)
data_train_full <- bind_rows(data_train, data_validation)
```

**Comment by hfrick** [comment]
Thanks for the request and nice to hear that it's fun to teach tidymodels! Have you seen `last_fit()` from tune for that final model fit? It has an argument `add_validation_set` which allows you to specify that both the training and validation set are used to train that final model. 
https://tune.tidymodels.org/reference/last_fit.html

**Comment by paulcbauer** [comment]
Haven't seen it and it solves the problem. Thanks!
---------------------------------

Dr. Paul C. Bauer

University of Freiburg (Political Science), LMU Munich (Statistics), MZES
Mannheim (External fellow)

Current paper(s): Who tweets and how freely?
<https://doi.org/10.1177/20531680221144237>, From ideal experiments to
ideal research designs (IDRs) <https://osf.io/4dahm/>, Can we predict
multi-party elections with Google Trends data?
<https://doi.org/10.1186/s40537-023-00868-4>
Email: ***@***.***
Websites: Homepage <http://www.paulcbauer.eu/>, Google Scholar
<https://scholar.google.ch/citations?user=hw-b-MkAAAAJ>, Twitter
<https://twitter.com/p_c_bauer>, Github <https://github.com/paulcbauer>


The information transmitted is intended only for the person or entity to
which it is addressed and may contain confidential and/or privileged
material. Any review, retransmission, dissemination, distribution,
forwarding, or other use of, or taking of any action in reliance upon, this
information by persons or entities other than the intended recipient is
prohibited without the express permission of the sender. If you received
this communication in error, please contact the sender and delete the
material from any computer.

**Comment by hfrick** [comment]
Excellent, closing this issue then!
